### PR TITLE
Update script to auto generate changelog

### PR DESCRIPTION
**Pull request by samirsilwal**
## Changes 
- Updated `release.sh` to auto generate changelog once the package is tagged

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -18,11 +18,13 @@ changelog() {
...
   echo "Generating changelog upto version: $NEXT"
   github_changelog_generator \
+    --user="leapfrogtechnology" \
+    --project="sync-db" \
+    --token="$GITHUB_TOKEN" \
     --no-verbose \
     --pr-label "**Changes**" \
     --bugs-label "**Bug Fixes**" \
     --issues-label "**Closed Issues**" \
-    --issue-line-labels=ALL \
```
Comment (samirsilwal) [inline release.sh:25]: @kabirbaidhya @mesaugat i was getting this error locally 
Screenshot from 2021-03-09 17-37-29
says notsuchmethod so i removed it. does it affect anything ?

File: release.sh [hunk 2]
```diff
@@ -69,6 +71,13 @@ compare_and_release() {
 
   if [ -n "$NEXT" ] && [ "$NEXT" != "false" ]; then
     bump
+    changelog
+
+    git add CHANGELOG.md
+    git commit -v --edit -m "${NEXT} Release :tada: :fireworks: :bell:"
+    git push origin master
+
+    printfln "Updated changelog."
```
Comment (mesaugat) [inline release.sh:80]:
```suggestion
    printfln "Changelog updated."
```

File: release.sh (hunk 1 above)
Comment (mesaugat) [reply release.sh:25]: It should be fine.

Comment (mesaugat) [reply release.sh:25]: We lost `labels` from the changelog but it seems to be better in my opinion. Feature not a bug.

File: release.sh [hunk 3]
```diff
@@ -69,6 +83,16 @@ compare_and_release() {
 
   if [ -n "$NEXT" ] && [ "$NEXT" != "false" ]; then
     bump
+    changelog
+
+    setup_git
```
Comment (mesaugat) [inline release.sh:88]: Why do we need a function? Can we put all the commands inline?

File: release.sh [hunk 4]
```diff
@@ -69,6 +83,16 @@ compare_and_release() {
...
+
+    git add CHANGELOG.md
+    git commit -v --edit -m "${NEXT} Release :tada: :fireworks: :bell:"
+
+    git_push
```
Comment (mesaugat) [inline release.sh:93]: Same.

**Comment by samirsilwal** [comment]
> @samirsilwal Shouldn't we commit the changelog first, tag the release and then deploy? We are doing deploy first.

um ok. sounds fair will move release.sh run in before deploy

File: release.sh (hunk 1 above)
Comment (samirsilwal) [reply release.sh:25]: OK resolving

**Comment by mesaugat** [comment]
@samirsilwal Please rebase. The other PR just got merged.

**Review by kabirbaidhya — APPROVED** [review]

File: release.sh [hunk 5]
```diff
@@ -68,6 +70,22 @@ compare_and_release() {
   fi
 
   if [ -n "$NEXT" ] && [ "$NEXT" != "false" ]; then
+    changelog
+
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "Travis CI"
+
+    git add CHANGELOG.md
+    git commit -v --edit -m "${NEXT} Release :tada: :fireworks: :bell:" -m "[skip ci]"
```
Comment (mesaugat) [inline release.sh:79]: @samirsilwal I used this. Much clean, such wow! 🙏 

https://stackoverflow.com/a/5070502/2352587

**Review by mesaugat — APPROVED** [review]

**Pull request merged**